### ISSUE TITLE
Inclusion in F-Droid 

### ISSUE THREAD

**Issue by RasheedAZ** [dluvian/Nozzle#67]
Hi. Thanks for making this beautiful app : )

 Are you open towards submitting it to the F-Droid main repo?

**Comment by dluvian** [comment]
I tried to create an issue in the [Requests For Packaging](https://gitlab.com/fdroid/rfp/issues) issue tracker but GitLab requires phone number verification which I don't want to do.

If there's an easy way to submit it without doing KYC let me know and I will try it.

**Comment by RasheedAZ** [comment]
Thanks for following up. That's strange... I have an account but it never asked for a phone number... Also, the signup page doesn't have a phone number field on the first step at least:
https://gitlab.com/users/sign_up

Were you logging in using the SSO options instead of email?

**Comment by dluvian** [comment]
Yes, I tried the GitHub SSO option which also claimed my email address so I had to create a new email just for trying the email sign up option. But the deed is done: https://gitlab.com/fdroid/rfp/-/issues/2650
I did not donate, because I don't have a bank account and they don't accept Bitcoin donations.
